home_x: 0
home_y: 0
home_z: 0
home_quat_x: 0
home_quat_y: 0
home_quat_z: 0
home_quat_w: 0
op_state: 'ready'
